THIRD-PARTY-LICENSES.txt

```diff
@@ -0,0 +1,386 @@
+     (Apache License, Version 2.0) Apache Commons Text (org.apache.commons:commons-text:1.6 - http://commons.apache.org/proper/commons-text)
+     (Apache License, Version 2.0) Apache Commons VFS (org.apache.commons:commons-vfs2:2.3 - http://commons.apache.org/proper/commons-vfs/)
+     (Apache License, Version 2.0) Apache HttpClient Fluent API (org.apache.httpcomponents:fluent-hc:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpAsyncClient (org.apache.httpcomponents:httpasyncclient:4.1.2 - http://hc.apache.org/httpcomponents-asyncclient)
+     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.10 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpClient Cache (org.apache.httpcomponents:httpclient-cache:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpClient OSGi bundle (org.apache.httpcomponents:httpclient-osgi:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpCore (org.apache.httpcomponents:httpcore:4.4.12 - http://hc.apache.org/httpcomponents-core-ga)
+     (Apache License, Version 2.0) Apache HttpCore NIO (org.apache.httpcomponents:httpcore-nio:4.4.12 - http://hc.apache.org/httpcomponents-core-ga)
+     (Apache License, Version 2.0) Apache HttpClient Mime (org.apache.httpcomponents:httpmime:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) tomcat-jdbc (org.apache.tomcat:tomcat-jdbc:9.0.16 - https://tomcat.apache.org/)
+     (Apache License, Version 2.0) tomcat-juli (org.apache.tomcat:tomcat-juli:9.0.16 - https://tomcat.apache.org/)
+     (Apache License, Version 2.0) AssertJ fluent assertions (org.assertj:assertj-core:3.14.0 - http://assertj.org/assertj-core)
+     (Apache License 2.0) Asynchronous Http Client (org.asynchttpclient:async-http-client:2.6.0 - http://github.com/AsyncHttpClient/async-http-client/async-http-client)
+     (Apache License 2.0) Asynchronous Http Client Netty Utils (org.asynchttpclient:async-http-client-netty-utils:2.6.0 - http://github.com/AsyncHttpClient/async-http-client/async-http-client-netty-utils)
+     (Bouncy Castle Licence) Bouncy Castle PKIX, CMS, EAC, TSP, PKCS, OCSP, CMP, and CRMF APIs (org.bouncycastle:bcpkix-jdk15on:1.59 - http://www.bouncycastle.org/java.html)
+     (Bouncy Castle Licence) Bouncy Castle PKIX, CMS, EAC, TSP, PKCS, OCSP, CMP, and CRMF APIs (org.bouncycastle:bcpkix-jdk15on:1.60 - http://www.bouncycastle.org/java.html)
+     (Bouncy Castle Licence) Bouncy Castle Provider (org.bouncycastle:bcprov-ext-jdk15on:1.54 - http://www.bouncycastle.org/java.html)
+     (Bouncy Castle Licence) Bouncy Castle Provider (org.bouncycastle:bcprov-jdk15on:1.60 - http://www.bouncycastle.org/java.html)
+     (Unknown license) cromwell-common (org.broadinstitute:cromwell-common_2.12:41 - no url defined)
```

Comment:
Need to investigate, there should be a way to specify this

Reply:
Now get:
(Cromwell License https://github.com/broadinstitute/cromwell/blob/develop/LICENSE.txt) (Unknown license) cromwell-wdl-transforms-shared (org.broadinstitute:cromwell-wdl-transforms-shared_2.12:41 - no url defined)

Reply:
According to documentation https://www.mojohaus.org/license-maven-plugin/examples/example-thirdparty.html that is expected result